lib/index.js

```diff
@@ -30,12 +29,7 @@ const exprGen = {
       `${exprGen[node.right.kind](node.right)})`
   },
   block: function (node) { // ['body' ],
-    let res = '(block'
-    node.body.forEach(function (e) {
-      res += ' ' + exprGen[e.kind](e)
-    })
-    res += ')'
-    return res
+    return `(block ${node.body.map((e) => exprGen[e.kind](e)).join(' ')})`
```

Comment:
@wanderer I think we should stick with ES5 for now.
Please use `function` instead of `=>`